test/ModalSpec.js

```diff
@@ -85,4 +85,16 @@ describe('Modal', function () {
     assert.ok(dialog.className.match(/\bmodal-sm\b/));
   });
 
+  it('Should pass dialogClassName to the dialog', function () {
+    let noOp = function () {};
+    let instance = ReactTestUtils.renderIntoDocument(
+      <Modal dialogClassName="testCss" onRequestHide={noOp}>
+        <strong>Message</strong>
+      </Modal>
+    );
+
+    let dialog = instance.getDOMNode().getElementsByClassName('modal-dialog')[0];
```

Comment:
Use of ReactTestUtils is preferred here: https://facebook.github.io/react/docs/test-utils.html#findrendereddomcomponentwithclass